src/PrestaShopBundle/Resources/views/Admin/Common/table.html.twig

```diff
@@ -0,0 +1,105 @@
+{% set filtersForm = tableView.formView %}
+
+{{ form_start(filtersForm) }}
+<table class="table" id="{{ tableView.identifier }}">
+    <thead class="thead-default">
+    <tr class="column-headers">
+        {% for tableColumn in tableView.columns %}
+            <th scope="col">
+                {% if tableColumn.is_sortable %}
+                    {{ ps.sortable_column_header(tableColumn.name, tableColumn.identifier, '', 'asc') }}
+                {% else %}
+                    {{ tableColumn.name }}
+                {% endif %}
+            </th>
+        {% endfor %}
+        <th scope="col">&nbsp;</th>
+    </tr>
+    {% if filtersForm|length > 1 %}
+    <tr class="column-filters">
+        {% for tableColumn in tableView.columns %}
+            <th>
+                {% if filtersForm[tableColumn.identifier] is defined %}
+                    {{ form_widget(filtersForm[tableColumn.identifier]) }}
+                {% else %}
+                    -
+                {% endif %}
+            </th>
+        {% endfor %}
+
+        <th class="text-right">
+            <button type="submit" class="btn btn-primary">
+                <i class="material-icons">search</i>{{ 'Search'|trans({}, 'Admin.Shopparameters.Feature') }}
+            </button>
+
+            {% if tableView.formView.vars.data is not empty %}
+                {#<a href="{{ path('admin_sql_manager', {'offset': filters.offset, 'limit': filters.limit}) }}"#}
```

Comment:
reset button url may be a  common action, see my PR about search system :) don't worry too much about it right now